doc/guides/west/release-notes.rst

```diff
@@ -1,6 +1,70 @@
 West Release Notes
 ##################
 
+v0.9.0
+******
+
+.. warning::
+
+   The ``west config`` fix described below comes at a cost: any comments or
+   other manual edits in configuration files will be removed when setting a
+   configuration option via that command or the ``west.configuration`` API.
+
+New features:
+
+- West manifests now support :ref:`west-manifest-submodules`. This allows you
+  to clone `Git submodules
+  <https://git-scm.com/book/en/v2/Git-Tools-Submodules>`_ into a west project
+  repository in addition to the project repository itself.
+
+- West manifests now support :ref:`west-manifest-groups`. Project groups can be
+  controlled via allowlists and blocklists to determine what projects are
```

Comment:
Still needs an update here, from `allow` and `block`.

How about:
```suggestion
  controlled via group keys to determine what projects are
```

Reply:
Nice catch.

I think I'll go with:

```
- West manifests now support :ref:`west-manifest-groups`. Project groups can be
  enabled and disabled to determine what projects are "active", [...]
```